components/org.wso2.micro.integrator.core/src/main/java/org/wso2/micro/integrator/core/internal/MicroIntegratorBaseConstants.java

```diff
@@ -46,6 +46,9 @@
     public static final String COMPONENT_REP0_ENV = "COMPONENTS_REPO";
     public static final String AXIS2_REPO_ENV = "AXIS2_REPO";
     public static final String UPDATE_LEVEL = "UPDATE_LEVEL";
+    public static final String TRANSPORT_PORT = "port";
+    public static final String TRANSPORT_HTTP = "http";
+    public static final String TRANSPORT_HTTPS = "https";
```

Comment:
Do we need to redefine these? Can't we use the existing constants?

Reply:
Fixed